shared/substrate/events.go

```diff
@@ -169,6 +169,18 @@ type EventRegistryTmp struct {
 	Topics []types.Hash
 }
 
+type EventDataStringSet struct {
+	Phase     types.Phase
+	AccountId types.AccountID
+	Topics    []types.Hash
+}
+
+type EventDataStringChanged struct {
```

Comment:
Can't we use the same type, since structure the same?

Reply:
let it be two different structs, so that in future it will be helpful to extend it if anything is required.